packages/xo-collection/src/collection.test.js

```diff
@@ -0,0 +1,356 @@
+import fromEvent from 'promise-toolbox/fromEvent'
+import forEach from 'lodash/forEach.js'
+
+import assert from 'assert'
+import { test, describe, beforeEach } from 'node:test'
+
+import { Collection, DuplicateItem, NoSuchItem } from './collection'
```

Comment:
Not all tests are following this principle, but I now think it's better to use public identifiers when possible because it validates exports as defined in `package.json`.

```suggestion
import { Collection, DuplicateItem, NoSuchItem } from 'xo-collection'
```